lib/rmt/cli/smt_importer.rb

```diff
@@ -0,0 +1,161 @@
+require 'csv'
+require 'ostruct'
+
+
+class SMTImporter
+  attr_accessor :config
+
+  class ImportException < StandardError
+  end
+
+  def initialize(config)
+    @config = config
+  end
+
+  def read_csv(file)
+    # set the quote char to something not used to make sure the csv parser is not interfering with the
+    # JSON quoting.
+    CSV.open(File.join(config.data_dir, file + '.csv'), 'r', { col_sep: "\t", quote_char: "\x00" })
+  end
+
+  def import_repositories
+    read_csv('enabled_repos').each do |row|
+      repo_id, = row
+      repo = Repository.find_by(scc_id: repo_id)
+      if repo
+        repo.mirroring_enabled = true
+        repo.save!
+        puts "Enabled mirroring for repository #{repo_id}"
+      else
+        warn "Repository #{repo_id}, perhaps you no longer have a valid subscription for it"
```

Comment:
`Repository #{repo_id}, perhaps ...` -> `Repository #{repo_id} was not found in RMT DB, perhaps ...`